### ISSUE TITLE
Requirements are outdated

### ISSUE THREAD

**Issue by Jamsy0707** [aminatouseyeup/Deepfakes#7]
Many of the packages listed won't install properly due to being outdated. I'm particularly having trouble with deep-animator as it relies on ConfigParser which, as I understand, is now defunct. Any ideas?

**Comment by aminatouseyeup** [comment]
Hi ! Indeed, several packages can be difficult to install if they are outdated. Regarding ConfigParser, it is important to note that this module is not obsolete and is still used to manage configuration files in Python. For deep-animator, if you encounter problems with specific dependencies, I recommend using the pip list command to check the versions of the packages installed in your environment. This will help you identify the packages that need to be updated or downgraded. Using a virtual environment can also help isolate and resolve these issues

**Comment by tedoul2107** [comment]
Hi @Jamsy0707, 

We've updated the readme.
Try to install for windows :
- Microsoft Visual C++ 14.0 or greater
- ffmpeg

and after try to install all requirements with a version of python 3.7.

If you still have problems installing packages, try installing them one by one in the right order.

**Comment by Jamsy0707** [comment]
This didn't work. I'm getting a ton of conflicting dependency errors now. I tried installing them one by one to see if I could resolve them but it doesn't seem possible. Also, I noticed in requirements.txt, it should be scikit-image==0.16.1, instead of sickit-image==0.16.

ERROR: Cannot install -r requirements.txt (line 1), -r requirements.txt (line 17), -r requirements.txt (line 20), -r requirements.txt (line 23) and numpy>=1.14.0 because these package versions have conflicting dependencies.

The conflict is caused by:
    The user requested numpy>=1.14.0
    numba 0.48.0 depends on numpy>=1.15
    imageio 2.3.0 depends on numpy
    tensorflow 1.15.0 depends on numpy<2.0 and >=1.16.0
    deep-animator 0.1.1 depends on numpy==1.15.0

**Comment by tedoul2107** [comment]
Yes @Jamsy0707, you're right, even at our level, we encountered a lot of difficulties during installation, mainly because of version dependency problems.

The only way that worked was to install the packages in the readme, respecting the order in the requirements.txt file.

The final requirement we obtained is the one attached in the discussion.

We'll try to find alternatives to make package installation less problematic. 
spolier with a docker image ;) 
[all-requirements.txt](https://github.com/aminatouseyeup/Deepfakes/files/15081254/all-requirements.txt)
